.github/workflows/push.yml

```diff
@@ -0,0 +1,118 @@
+name: push-pipeline
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '**'
+
```

Comment:
One small suggestion here: 
- We can define the repo-name, image-name, and image-tag as global ENV and use them inside the workflow so that we don't have to change at every place when have to change the values.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env

Reply:
Okay. It can do done